.github/workflows/push-builder.yml

```diff
@@ -0,0 +1,22 @@
+name: build-image
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
Depending on branch `main` requires #2677 to be merged before this PR.

Reply:
I am ok with this depending on `master` and rebasing 2677 on this PR if you would like to change this back, @mauilion